api/src/uvisor-input.S

```diff
@@ -117,7 +117,7 @@ __uvisor_box_index_size:
     /* default index size is 64B - user can override weak reference */
     .long 64
 
-.section .bss
+.section .uninitialized
```

Comment:
Add comments in the code and/or the commit description explaining why moving from `.bss` to `.uninitialized` is beneficial.